submissions/2024-10-08-OpenFF-Lipid-Optimization-Training-Supplement-v1.0/README.md

```diff
@@ -0,0 +1,67 @@
+# OpenFF Lipid Optimization Training Supplement v1.0
+
+## Description
+
+An optimization data set created to improve the training coverage of lipid-like
+molecules in Sage. The molecules in this data set were selected from the [LIPID
+MAPS](https://www.lipidmaps.org/) database via
+[cura](https://github.com/ntBre/curato/tree/6039bea5c64f8cd6b374fd12b5fa3b355898d98b)
+after being fragmented by the [XFF
+fragmentation](https://github.com/XtalPi-XFF/2023_XFF_paper/tree/b84ef6079d24ebd2e86f78a495e3257b375255fa/fragmentation)
+algorithm, clustered on the 2048-bit, radius-3 Morgan fingerprint from RDKit by
+the `LeaderPicker.LazyBitVectorPick` algorithm, also from RDKit, with a distance
+threshold of 0.6559. Placeholder dummy atoms in the fragments were generally
+filled with H atoms, and `S(=O)(=O)*` groups were replaced with `S(=O)([O-])`.
+The candidate fragments were further restricted to those with between 3 and 70
+heavy atoms and containing only the elements Cl, P, Br, I, H, C, O, N, F, and S.
+Candidates with InChIKeys matching existing Sage training or benchmarking data
+were also filtered out.
+
+## General Information
+
+* Date: 2024-10-08
+* Class: OpenFF Optimization Dataset
+* Purpose: Improve training coverage in Sage
+* Name: OpenFF Lipid Optimization Training Supplement v1.0
+* Number of unique molecules: 3971
+* Number of filtered molecules: 32
```

Comment:
Just checking -- are these number of filtered molecules the ones that were filtered *out*, i.e. are not part of the number of unique molecules above?

Reply:
Yes that's right. This number is usually zero (at least in my previous submissions), but this time qcsubmit actually filtered out 32 molecules, I think for timeouts in conformer generation. My input SMILES file had 4003 entries, so these add up to that total at least.

Reply:
Thanks Brent!